src/pages/settings/Subscription/SubscriptionPlan.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {Linking, Platform, View} from 'react-native';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useSubscriptionPlan from '@hooks/useSubscriptionPlan';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+function SubscriptionPlan() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    const subscriptionPlan = useSubscriptionPlan();
+
+    const isCollect = subscriptionPlan === CONST.POLICY.TYPE.TEAM;
+    // TODO: replace this with a real value once OpenSubscriptionPage API command is implemented
+    const isAnnual = true;
+
+    const benefitsList = isCollect
```

Comment:
try something like this - just insert key - instead of prepare arrays 2 times
const translateKey = isCollect ? 'collect' : 'control';
const benefitsList = `[ translate('subscription.yourPlan.${translateKey}.benefit1'),]`